src/Utilities/Constants.js

```diff
@@ -1,2 +1,8 @@
 export const DEFAULT_PAGE_SIZE = 50;
+export const SOURCES_API_BASE = '/r/insights/platform/topological-inventory/v0.0';
```

Comment:
const topo_apiHost = process.env.TOPO_API_HOST || '';
const topo_apiPort = process.env.TOPO_API_PORT || '443';
const topo_basePath = process.env.TOPO_BASE_PATH;
const topo_apiVersion =  process.env.TOPO_API_VERSION || '/api/v0.0';
export const SOURCES_API_BASE = `https://'${topo_apiHost}:${topo_apiPort}${topo_basePath}${topo_apiVersion.replace(/\/+$/, '')}`;

Reply:
> const topo_apiHost = process.env.TOPO_API_HOST || '';
> const topo_apiPort = process.env.TOPO_API_PORT || '443';
> const topo_basePath = process.env.TOPO_BASE_PATH;
> const topo_apiVersion = process.env.TOPO_API_VERSION || '/api/v0.0';
> export const SOURCES_API_BASE = `https://'${topo_apiHost}:${topo_apiPort}${topo_basePath}${topo_apiVersion.replace(/\/+$/, '')}`;

Ok after small discussion with @karelhala we ought not to use full URI path for our APIs. host port etc. is retrieved from origin evrionment because API can run on multiple servers. 
There are 3 envirnoments where the APIs live: CI, QA, Prod. If we hard code base path from host we will break the UI on other server.

Also the base path you've suggested is missing some fragments of the URL

The reason why is it working for the service portal API is that we are running it locally.

> Unfortunatelly there seems to be some issue with the topological API so i've added wip label until i figure out what it is.

After i've removed your suggestion it is working OK.